### DISCUSSION TITLE
How to visual the GitHub repo's branches?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [community/community#45783] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

such as there is a repo has several branches:
![image](https://user-images.githubusercontent.com/12727582/215464896-9d384bf4-8eea-4cd8-83dd-9a6158d0163c.png)

is there a way to visual the git branches like below in GitHub?
![image](https://user-images.githubusercontent.com/12727582/215465230-3a234d3f-2069-4cf2-a01e-455b3bef2815.png)

**Comment by jge162** [comment]
Hello @liaodalin19903 No, GitHub does not currently offer a visual representation of branches like the one in your image. However, you can view the branches in a list format like first picture.

**Reply by liaodalin19903** [reply]
but in first picture how to compare which branch is ahead and which is behind?
